docs/upgrade-to-v3.asciidoc

```diff
@@ -0,0 +1,47 @@
+[[upgrade-to-v3]]
+
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/nodejs/current/upgrade-to-v3.html[elastic.co]
+endif::[]
+
+=== Upgrade to v3.x
+
+The following is a guide on upgrading your Node.js agent from version 2.x to version 3.x.
+
+[[v3-overview]]
+==== Overview
+
+Version 3.0.0 of the Node.js agent supports Node.js v8 (from v8.6.0 and onwards), v10, and v12.
+
+[[v3-config-options]]
+==== Config options
+
+The <<disable-instrumentations,`disableInstrumentations`>> config option now behaves differently if given the values `http` and/or `https`.
+Previously this would disable tracing of incoming and outgoing requests.
+Now this config option only deals with outgoing requests.
+To disable tracing of incoming http(s) requests, use the new <<instrument-incoming-http-requests,`instrumentIncomingHTTPRequests`>> config option.
+
+It's now possible to make use of manual instrumention while the <<instrument,`instrument`>> config option is set to `false`.
+This means that calls to for instance <<apm-start-transaction,`apm.startTransaction()`>> or <<apm-start-span,`apm.startSpan()`>> will produce transactions and spans even if `instrument` is set to `false`.
+
+[[v3-api-changes]]
+==== API changes
+
+The `type` associated with transactions and spans is no longer dot-separated.
+Instead the `type` property have been split into three distinct properties: `type`, `subtype`, and `action`.
```

Comment:
```suggestion
Instead the `type` property has been split into three distinct properties: `type`, `subtype`, and `action`.
```

I remember you mentioning have/has at one point to me. Here's a bit of info in case you're curious. Feel free to skip if you'd like 😃 

* **Have** is used when you're referring to something in the first, second, or third person plural:
  * 1st: I have / We have
  * 2nd: You have
  * 3rd plural: They have
* **Has** is used when speaking in the third person _singular_.
  * 3rd singular: He has, she has, it has

For this one, we're talking about the "`type` property" -- which would be a singular third person object -- it. So the sentence "the `type` property has been split into three distinct properties" could be reworded to somthing like  "it *has* been split into three distinct properties".

**A quick trick then would be:**
I / you / they -- Have
He / she / it -- Has

Reply:
Thank you ❤️